app/settings.go

```diff
@@ -31,6 +31,7 @@ type settings struct {
 	propertyLock   sync.RWMutex
 	theme          fyne.Theme
 	themeSpecified bool
+	mode           fyne.ThemeVariant
```

Comment:
```suggestion
	variant           fyne.ThemeVariant
```

Reply:
I was trying to avoid duplication...

Reply:
Hmm, I think adding this another, similar term confuses more that the duplication would cause stutter, besides there is already `theme fyne.Theme` above

Reply:
fixed